compiler/src/dotty/tools/dotc/ast/Positioned.scala

```diff
@@ -23,8 +23,9 @@ abstract class Positioned(implicit @constructorOnly src: SourceFile) extends Pro
    */
   def uniqueId: Int = myUniqueId
 
-  def uniqueId_=(id: Int): Unit = {
+  private def uniqueId_=(id: Int): Unit = {
     //assert(id != 2523, this)
```

Comment:
Maybe we should designate with a comment that these commented lines (and everywhere else) are intended for debug reasons and not accidental leftovers.

Reply:
Done

Reply:
CAPS LOCK? It looks like shouting 🤣 

APPROVED